test-case-reporting/redacted.env

```diff
@@ -2,3 +2,4 @@ DB_UNIX_SOCKET=/cloudsql/amp-test-cases:us-central1:amp-test-results
 DB_USER=[REDACTED]
 DB_PASSWORD=[REDACTED]
 DB_NAME=test_results
+TRAVIS_API_TOKEN=[REDACTED]
```

Comment:
If this isn't used yet, let's drop it here.

Reply:
Done.